stellargraph/layer/gcn_lstm.py

```diff
@@ -67,10 +68,11 @@ def __init__(
         if "input_shape" not in kwargs and input_dim is not None:
             kwargs["input_shape"] = (input_dim,)
 
-        self.units = (units,)
+        self.units = units  # (units,)
```

Comment:
i think the comment can be removed here as well
```suggestion
        self.units = units
```